atomic_reactor/util.py

```diff
@@ -649,18 +649,19 @@ def default(self):
         return self.v2 or self.v1
 
 
-def get_manifest_digests(image, registry, insecure=False, dockercfg_path=None,
-                         versions=('v1', 'v2')):
+def query_registry(image, registry, digest=None, insecure=False, dockercfg_path=None,
```

Comment:
Currently we make 3 requests to the registry - 2 for digests and 1 for config. I'll try to come with a way to optimize this, but I think this is acceptable for now